easybuild/easyconfigs/d/dftd3-lib/dftd3-lib-0.10-iccifort-2020.4.304.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'dftd3-lib'
+version = '0.10'
+
+homepage = 'https://github.com/dftbplus/dftd3-lib'
+description = """This is a repackaged version of the DFTD3 program by S. Grimme and his coworkers.
+ The original program (V3.1 Rev 1) was downloaded at 2016-04-03. It has been
+ converted to free format and encapsulated into modules."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.4.304'}
+toolchainopts = {'pic': True}
+
+github_account = 'dftbplus'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+patches = ['dftd3-lib-%(version)s_fix-extras-syntax.patch']
```

Comment:
Why not use the existing dftd3-lib-0.9_fix-extras-syntax.patch

Reply:
ah yeah that should work, they are the same :)